src/pretix/control/forms/filter.py

```diff
@@ -1116,9 +1116,31 @@ class SubEventFilterForm(FilterForm):
     )
 
     def __init__(self, *args, **kwargs):
+        self.organizer = kwargs.pop('organizer')
+        self.event = kwargs.pop('event')
         super().__init__(*args, **kwargs)
         self.fields['date_from'].widget = DatePickerWidget()
         self.fields['date_until'].widget = DatePickerWidget()
+        seen = set()
+        for p in self.meta_properties.all():
+            if p.name in seen:
+                continue
+            seen.add(p.name)
+            self.fields['meta_{}'.format(p.name)] = forms.CharField(
+                label=p.name,
+                required=False,
+                widget=forms.TextInput(
+                    attrs={
+                        'data-typeahead-url': reverse('control:event.subevents.meta.typeahead', kwargs={
+                            'organizer': self.organizer.slug,
+                            'event': self.event.slug
+                        }) + '?' + urlencode({
+                            'property': p.name,
+                            'organizer': self.organizer.slug if self.organizer else ''
```

Comment:
```suggestion
                            'organizer': self.organizer.slug if self.organizer else ''
```
pointless if the typeahead url is event-scoped

Reply:
I think you meant removing the if-else-part? That is right, pointless. I will remove it.

Reply:
No, removing the entire line. Why do you need the organizer as the query argument when it's already in the URL?

Reply:
Oh, sorry! Totally right.